testcases/kernel/syscalls/ipc/msgrcv/msgrcv08.c

```diff
@@ -47,11 +47,13 @@ const char *TCID = "msgrcv08";
 const int TST_TOTAL = 1;
 
 #if __WORDSIZE == 32
-
+#ifdef __GLIBC__
 struct msgbuf {
 	long mtype;     /* message type, must be > 0 */
 	char mtext[16]; /* message data */
 };
+#else
+#endif
 
```

Comment:
Based on latest ltp upstream, I am not define _GNU_SOURCE in msgrcv08.c and its' include files, that is strange.